### PR TITLE
Radio button

### PR DESCRIPTION
**Pull request by hiteshpr**

### CODE REVIEW

**Comment by hiteshpr** [comment]
Fixes [#35](https://github.com/jajodiaraghav/rTunes/issues/35)

File: src/app/components/search/search.component.ts [hunk 1]
```diff
@@ -66,6 +66,9 @@ export class SearchComponent {
         this.pagedItems = this.searchRes.slice(this.pager.startIndex, this.pager.endIndex + 1);
 	}
 	searchBasedOnCategory(category: string) {
+
+		this.searchbar = false;
```
Comment (jajodiaraghav) [inline src/app/components/search/search.component.ts]: Is it really working?
The searchbar variable should change to false only when a radio button is selected.

Can you please provide a gif?

Comment (hiteshpr) [reply src/app/components/search/search.component.ts]: searchBasedOnCategory() function is called on click of any of the radio button, so yes the code is working.

Comment (jajodiaraghav) [reply src/app/components/search/search.component.ts]: Cool

**Comment by hiteshpr** [comment]
@jajodiaraghav need GIF?

**Pull request merged**